tests/config_tests.py

```diff
@@ -13,6 +13,12 @@ def setUp(self):
         realpath = os.path.dirname(os.path.realpath(__file__))
         os.environ['HOME'] = os.path.join(realpath, 'fixtures')
 
+    def test_config_no_config(self):
+        cfg = Config()
```

Comment:
isn't it overiden by `cfg = Config('mackup-empty.cfg')` in `test_config_empty()`? Don't worry to copy paste in unit tests, that's fine.